src/controllers/imgurController.js

```diff
@@ -0,0 +1,83 @@
+const { ImgurClient } = require('imgur');
+const User = require('../models/User');
+
+const imgurClient = new ImgurClient({
+  clientId: process.env.IMGUR_CLIENT_ID,
+  clientSecret: process.env.IMGUR_CLIENT_SECRET,
+  refreshToken: process.env.IMGUR_REFRESH_TOKEN,
+});
+
+const renderUploadPage = (req, res, message, imageUrl) => {
+  const userId = req.params.userId;
+  res.render('imgur', { userId, message, imageUrl });
```

Comment:
When the user first loads the page by just navigating to `http://localhost:8000/:userId/uploadImage` they won't have a message or image url yet because they haven't yet tried to upload anything. So we can just set those variables to be empty:

```suggestion
  res.render('imgur', { userId, message: null, imageUrl: null });
```